packages/theme/src/components/index.ts

```diff
@@ -1,38 +1,38 @@
-import Accordion, { accordionStyles } from "./accordion"
-import Alert, { alertStyles } from "./alert"
-import Avatar, { avatarStyles } from "./avatar"
-import Badge, { badgeStyles } from "./badge"
-import Button, { buttonStyles } from "./button"
-import Checkbox, { checkboxStyles } from "./checkbox"
-import CloseButton, { closeButtonStyles } from "./close-button"
-import Code, { codeStyles } from "./code"
-import Modal, { modalStyles } from "./modal"
-import Drawer, { drawerStyles } from "./drawer"
-import Heading, { headingStyles } from "./heading"
-import Input, { inputStyles } from "./input"
-import Link, { linkStyles } from "./link"
-import FormLabel, { formLabelStyles } from "./form-label"
-import Menu, { menuStyles } from "./menu"
-import NumberInput, { numberInputStyles } from "./number-input"
-import Radio, { radioStyles } from "./radio"
-import Slider, { sliderStyles } from "./slider"
-import Select, { selectStyles } from "./select"
-import Spinner, { spinnerStyles } from "./spinner"
-import Switch, { switchStyles } from "./switch"
-import Tabs, { tabsStyles } from "./tabs"
-import Tag, { tagStyles } from "./tag"
-import Tooltip, { tooltipStyles } from "./tooltip"
-import Kbd, { kbdStyles } from "./kbd"
-import PinInput, { pinInputStyles } from "./pin-input"
-import Popover, { popoverStyles } from "./popover"
-import Form, { formStyles } from "./form"
-import Editable, { editableStyles } from "./editable"
-import Progress, { progressStyles } from "./progress"
-import Textarea, { textareaStyles } from "./textarea"
-import Stat, { statStyles } from "./stat"
-import Skeleton, { skeletonStyles } from "./skeleton"
-import Breadcrumb, { breadcrumbStyles } from "./breadcumb"
-import SkipLink, { skipLinkStyles } from "./skip-link"
+import { default as Accordion } from "./accordion"
```

Comment:
if we plan on aliasing anyways, we should consider using named exports in the first place

Reply:
Done!